lib/bolt/transport/lxd.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'bolt/logger'
+require 'bolt/node/errors'
+require 'bolt/transport/simple'
+
+module Bolt
+  module Transport
+    class LXD < Simple
+      def provided_features
+        ['shell']
+      end
+
+      def with_connection(target, options = {})
+        Bolt::Logger.warn_once("lxd_experimental",
+                               "The LXD transport is experimental, and may change between minor versions.")
```

Comment:
```suggestion
                               "The LXD transport is experimental, and may include breaking changes between minor versions.")
```

Just to be a bit more explicit about what might happen.